AnimateQuotes.js

```diff
@@ -1,21 +1,57 @@
 import React, { useRef, useEffect, useState } from 'react';
 import { Animated, Text, View } from 'react-native';
-import { useIsShake } from './useIsShake';
-import { QUOTES_IN_DURATION, QUOTES_IN_DELAY, QUOTES_OUT_DURATION, SVG_IN_DELAY, SVG_IN_DURATION, SVG_OUT_DURATION, TRANSITION_TIME_2 } from './constants';
+import { QUOTES_IN_DURATION, QUOTES_IN_DELAY, QUOTES_OUT_DURATION, quotes } from './constants';
+import { getRandomInt } from './utils';
+import Quotes from "./Quotes";
 
-export default function AnimateQuotes(props) {
-  const [isRegistered, setIsRegistered] = useState(false);
-  const [hasToggledBefore, setHasToggledBefore] = useState(false);
+export default function AnimateQuotes({ isShakeTriggered, setIsShakeReady, hasInitialTransitionRun, setHasInitialTransitionRun }) {
+  const [isQuoteAnimationRunning, setIsQuoteAnimationRunning] = useState(false);
+  const [quoteIndex, setQuoteIndex] = useState(0);
 
   const fadeAnim = useRef(new Animated.Value(0)).current;
   const sizeAnim = useRef(new Animated.Value(0.5)).current;
 
-  const { toggle } = useIsShake();
+  function quoteAnimationIn() {
+    Animated.parallel([
+      Animated.timing(fadeAnim, {
+        toValue: 1,
+        duration: QUOTES_IN_DURATION,
+        delay: QUOTES_IN_DELAY,
+        useNativeDriver: true,
+      }),
+      Animated.timing(sizeAnim, {
+        toValue: 1,
+        duration: QUOTES_IN_DURATION,
+        delay: QUOTES_IN_DELAY,
+        useNativeDriver: true,
+      }),
+    ]).start(() => {
+      setIsQuoteAnimationRunning(false);
+      setIsShakeReady(true);
+      setHasInitialTransitionRun(true);
+    });
```

Comment:
FYI, the `finished` prop lets us know that the animation was not interrupted with a `stop()`. Don't think that this applies here, but just an FYI https://reactnative.dev/docs/animated#working-with-animations